src/patternfly/demos/DataToolbar/examples/DataToolbar.md

```diff
@@ -0,0 +1,524 @@
+---
+title: Data Toolbar
+section: demos
+---
+
+import './DataToolbar.css'
+
+## Demos
+```hbs title=Data-toolbar-attribute-value-search-filter-desktop
+{{#> data-toolbar data-toolbar--id="data-toolbar-attribute-value-search-filter-desktop"}}
+  {{#> data-toolbar-content}}
+    {{#> data-toolbar-group data-toolbar-group--modifier="pf-m-toggle-group pf-m-show-on-lg"}}
+      {{> data-toolbar-toggle data-toolbar-toggle--IsExpanded="false"}}
+      {{#> data-toolbar-group data-toolbar-group--modifier="pf-m-filter-group"}}
+        {{#> data-toolbar-item}}
+          {{#> input-group}}
+            {{#> select select--attribute="style='width: 150px'" id=(concat data-toolbar--id '-select-name') select-toggle--icon="fas fa-filter"}}
+              Name
+            {{/select}}
+            {{#> form-control controlType="input" input="true" form-control--attribute=(concat 'id="' data-toolbar--id '-textInput11" name="textInput11" type="search" placeholder="Filter by name..." aria-label="search input example"')}}
+            {{/form-control}}
+            {{#> button button--modifier="pf-m-control" button--attribute='aria-label="search button for search input"'}}
+              <i class="fas fa-search" aria-hidden="true"></i>
+            {{/button}}
+          {{/input-group}}
+        {{/data-toolbar-item}}
+      {{/data-toolbar-group}}
+    {{/data-toolbar-group}}
+    {{#> overflow-menu overflow-menu-button--aria-label="Expand overflown menu" overflow-menu--id=(concat data-toolbar--id '-overflow-menu') overflow-menu--modifier="pf-m-show-on-xl" overflow-menu-dropdown-button-aria-expanded="true"}}
+      {{#> overflow-menu-content}}
+        {{#> overflow-menu-item}}
+          {{#> data-toolbar-group data-toolbar-group--modifier="pf-m-button-group"}}
+            {{#> data-toolbar-item}}
+              {{#> button button--modifier="pf-m-primary"}}
+                Action
+              {{/button}}
+            {{/data-toolbar-item}}
+            {{#> data-toolbar-item}}
+              {{#> button button--modifier="pf-m-link"}}
+                Action
+              {{/button}}
+            {{/data-toolbar-item}}
+            {{#> data-toolbar-item}}
+              {{#> button button--modifier="pf-m-plain" button--attribute='aria-label="Actions"'}}
+                <i class="fas fa-ellipsis-v" aria-hidden="true"></i>
+              {{/button}}
+            {{/data-toolbar-item}}
+          {{/data-toolbar-group}}
+        {{/overflow-menu-item}}
+      {{/overflow-menu-content}}
+      {{#> overflow-menu-control}}
+        {{#> overflow-menu-dropdown-item}}
+          Action
+        {{/overflow-menu-dropdown-item}}
+        {{#> overflow-menu-dropdown-item overflow-menu-dropdown-item--IsShared="true"}}
+          Item 1
+        {{/overflow-menu-dropdown-item}}
+        {{#> overflow-menu-dropdown-item overflow-menu-dropdown-item--IsShared="true"}}
+          Item 2
+        {{/overflow-menu-dropdown-item}}
```

Comment:
`pf-m-shared` was removed from the overflow menu component as React moves these elements in the DOM. You can remove these `items`. 

https://patternfly-next-pr-2328.surge.sh/documentation/core/experimental/overflowmenu#persistent-additional-options-hidden is a good example to reference